examples/testthing/testclient.js

```diff
@@ -35,11 +47,23 @@ async function testPropertyWrite(thing, name, value, shouldFail) {
         if (!shouldFail) console.info("PASS " + name + " WRITE (" + displayValue + ")");
         else console.error("FAIL " + name + " WRITE: (" + displayValue + ")");
     } catch (err) {
-        if (!shouldFail) console.error("FAIL " + name + " WRITE (" + displayValue + "):", err.message);
-        else console.info("PASS " + name + " WRITE (" + displayValue + "):", err.message);
+        if (!shouldFail) {
+            if (err instanceof Error) {
+                console.error("FAIL " + name + " WRITE (" + displayValue + "):", err.message);
+            } else {
+                console.error("FAIL " + name + " WRITE (" + displayValue + "):", err);
+            }
```

Comment:
Not sure if we just use 

`console.error("FAIL " + name + " WRITE (" + displayValue + "):", err);`

in both cases... I don't think it is worth it..

Reply:
simplified in https://github.com/eclipse-thingweb/node-wot/pull/1049/commits/b23642209116bb8b0a9ed3be2fe7f1e7c8969c37